gwdetchar/lasso/__main__.py

```diff
@@ -267,6 +269,17 @@ def _process_channel(input_):
     return (chan, lassocoef, plot4, plot5, plot6, ts)
 
 
+def get_primary_ts(channel, start, end, filepath=None, frametype=None, cache=None, nproc=1):
```

Comment:
I suggest adding some documentation and/or other comment to explain what this function is doing, even if it is straightforward